client/app/bundles/course/survey/pages/ResponseShow/index.jsx

```diff
@@ -110,7 +110,7 @@ class ResponseShow extends React.Component {
 
   renderUnsubmitButton() {
     const { response, flags: { canUnsubmit, isLoading } } = this.props;
-    if (!canUnsubmit || isLoading) { return null; }
+    if (!canUnsubmit || isLoading || !response.submitted_at) { return null; }
```

Comment:
Just a quick check, typically you do your logical checks on whether to render the component or not in the `renderxxx` method itself?